website/events/api/calendarjs/views.py

```diff
@@ -56,7 +57,7 @@ def get_queryset(self):
 class CalendarJSUnpublishedEventListView(ListAPIView):
     """Define a custom route that outputs the correctly formatted external events information for CalendarJS, unpublished events only."""
 
-    queryset = ExternalEvent.objects.filter(published=False)
+    queryset = Event.objects.filter(published=False)
```

Comment:
This is the actual required bugfix @pingiun

Reply:
I think this something with the rebase that went wrong when fixing conflicts

Reply:
The rest is technically not required to fix the bug and get back to original functionality

Reply:
I think we should only have this one line fix then

Reply:
Correct that this is the fix, but I did not want to remove unpublished external events from the calendar, hence the rest of the PR.

Reply:
The rest of the PR is untested code, I just want to merge the fix so we can release on time